design.md

```diff
@@ -193,25 +215,35 @@ The client must periodically call a keepalive function in the kernel. If the ker
 
 #### Published Artifacts
 
-! RFC: Refine this
-
-- A shadow Jar that is cross-platform for desktop-class Windows, macOS, and Linux.
-- A shadow Jar for each embedded computer we support.
-- An image for each embedded computer we support.
-- deb and rpm packages that install the auto-updater for the kernel.
+- Embedded systems we support:
+  - Raspi
+  - Jetson nano
+- Shadow Jars of the kernel and the display that are cross-platform for desktop-class Windows, macOS, and Linux.
+- A shadow Jar of the kernel for each embedded computer we support.
+- An image containing the installed kernel and JVM for each embedded computer we support.
+  - It should come with a systemd service added but disabled.
```

Comment:
Why disabled by default in the image? Is the presumed use-case for least-advanced embedded users (i.e. the users we are targeting the image at) not to have it always running?

I'm all for having it be optionally enabled in the DEB/RPM packages, but in the preinstalled image that seems like it would just create frustration for new users.

Reply:
You are right, it should be enabled by default in the image.

I removed the deb and rpm packages because they were not necessary. If you are installing them, then you have access to the embedded computer in some form (and it has internet) so you could just grab the kernel Jar yourself. From there on, you would just use the IDE to update the kernel (making a separate package unnecessary).

Reply:
Hmm, the reason I was pushing for DEB and RPM packages was to make that install easier and well-defined (both for us in creating the images, and for the user in setting up their own). I don't think any of this removes that motivation. Obviously the user could just grab the JARs themselves, but that's not the problem I was aiming to solve.